### PR TITLE
Add more plugins to the plugins section

### PR DESCRIPTION
**Pull request by lipis**
Just to give a bit more exposure to some of the other plugins.. I'm aware of the fact that they might be far from done, but promoting them from here could attract more people.

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ If the plugin is unable to be found automatically, you can load them with:
 * [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
 * [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
 
+## Unofficial Plugins
```
Comment (lipis) [inline docs/plugins.md]: Other ideas for the title are more than welcome

File: docs/plugins.md [hunk 2]
```diff
@@ -38,6 +38,12 @@ If the plugin is unable to be found automatically, you can load them with:
...
+
+* [@thorbenvh8/prettier-java](https://github.com/thorbenvh8/prettier-java)
```
Comment (azz) [inline docs/plugins.md]: Please use the npm package names here.

Comment (lipis) [reply docs/plugins.md]: should I just remove the `@` or what?

the following work..

```
yarn add thorbenvh8/prettier-java
# or
npm install --save-dev thorbenvh8/prettier-java
```

Comment (azz) [reply docs/plugins.md]: `prettier-plugin-java`: https://www.npmjs.com/package/prettier-plugin-java

Comment (lipis) [reply docs/plugins.md]: the other two are not on NPM and for the same reason even Python is not correct.. (https://www.npmjs.com/package/@prettier/python)

We can just point to the GitHub repositories until they are actually released, as we are looking for early adopters/testers and not the actual users.

Comment (azz) [reply docs/plugins.md]: Yes, like to GitHub, but use the npm package names as the names.

File: docs/plugins.md [hunk 3]
```diff
@@ -38,6 +38,12 @@ If the plugin is unable to be found automatically, you can load them with:
 * [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
 * [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
 
+## Unofficial Plugins
+
+* [prettier-plugin-java](https://github.com/thorbenvh8/prettier-java)
+* [@iamsolankiamit/prettier-ruby](https://github.com/iamsolankiamit/prettier-ruby)
```
Comment (azz) [inline docs/plugins.md]: The `@`s are still incorrect as it implies it is an npm package name. Either `iamsolankiamit/prettier-ruby` or `prettier-plugin-ruby` is fine.

Comment (lipis) [reply docs/plugins.md]: Really confused.. :| Is this fine now?

File: docs/plugins.md (hunk 1 above)
Comment (suchipi) [reply docs/plugins.md]: "Community Plugins" might be a bit more friendly

Comment (lipis) [reply docs/plugins.md]: Much better!

**Comment by lipis** [comment]
@azz?

**Pull request merged**